### DISCUSSION TITLE
[Doubt] About using `field`.

### DISCUSSION THREAD

**Discussion by MuhammedZakir** [tree-sitter/tree-sitter#1795] [Q&A] [answered]
Consider this grammar:
```
heading: $ => seq(
  "#",
  repeat1(" "),
  alias($._line, $.content)
)
```

Should I use `alias` or `field` in the last sub-rule?

If, instead it was `$.content` rather than `$._line`, is there any reason or situation for me to do `field('content', $.content)`, or `field('heading', $.content)`?

**Comment by ahelwer** [comment, marked as answer]
It's a judgement you can make about which would be easiest to write queries for.

**Reply by MuhammedZakir** [reply]
I guess I will add it then. Thank you!
